### DISCUSSION TITLE
Strange results with pypy36 and pypy37

### DISCUSSION THREAD

**Discussion by brechtm** [ymyzk/tox-gh-actions#162] [Q&A] [answered]
I want to test my project against both PyPy 3.6 and 3.7. My _tox-gh-actions_ is set up in my _tox.ini_ like this:
```
[gh-actions]
python =
    3.6: py36
    3.7: py37
    3.8: py38
    3.9: py39
    3.10: py310
    pypy-3.6: pypy36
    pypy-3.7: pypy37
```

Here's a run of my GitHub Actions workflow: https://github.com/brechtm/rinohtype/actions/runs/515676468. For the _pypy-3.6 unit tests_ jobs on Ubuntu and macOS, **tox-gh-actions produces an empty envlist**. The _pypy-3.7_ job works correctly (``['pypy37']`` envlist).

On Windows, the PyPy jobs actually run on CPython 3.9. But I think that's not an issue with tox-gh-actions since the Poetry venv is created using CPython 3.9 for some reason.

While the _pypy36_ and _pypy37_ envs aren't explicitly listed in the tox docs, they do seem to do the right thing when I run tox locally. I have verified this with PyPy 3.7 on macOS. I don't have PyPy 3.6 installed, but at least tox is attempting to run the correct interpreter:
```
$ tox -e pypy36
pypy36 create: /Users/brechtm/Documents/Code/rinohtype/.tox/pypy36
ERROR: InterpreterNotFound: pypy3.6
_____________________________________________ summary ______________________________________________
ERROR:  pypy36: InterpreterNotFound: pypy3.6
```

**Comment by brechtm** [comment]
I may just have misunderstood how tox-gh-actions works. After discovering #44, I think I may need to include pypy36 in _envlist_? However, I would rather not, since I don't have pypy3.6 installed locally...

**Comment by ymyzk** [comment, marked as answer]
@brechtm, thanks for filing an issue. As you found in https://github.com/ymyzk/tox-gh-actions/issues/44#issuecomment-768963559, this is the current expected behavior of tox-gh-actions. Users are expected to add list of environments to run to `envlist`. Use of `skip_missing_interpreters` option provided by tox can be one workaround for local environment without all interpreters.
